subsys/net/conn_mgr/conn_mgr.c

```diff
@@ -31,118 +31,126 @@ K_SEM_DEFINE(conn_mgr_event_signal, 1, 1);
 /* Used to protect conn_mgr state */
 K_MUTEX_DEFINE(conn_mgr_lock);
 
-static enum net_conn_mgr_state conn_mgr_iface_status(int index)
```

Comment:
"Also rename a few flags." no longer seems to apply to commit 003a420e804be12170241674ab8a351f2de42492?

Reply:
Good catch! You're right, I chose to do that in a separate commit to help make reviewing easier. I forgot I had mentioned it in the commit message